app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -5,7 +5,7 @@ module TextualSummary
     #
 
     def textual_group_properties
-      %i(name tag id full_name)
+      %i(name tag id full_name os_distribution kernel_version)
```

Comment:
@zakiva 

Add `product_type` and `product_name`
Remove `kernel_version` which is useless for container images (as they're using the host kernel).